XXX disable dot(1) because it can break when run under dpb(1).

e.g.
/usr/local/bin/gi-docgen generate --quiet --fatal-warnings \
	--config doc/libgweather.toml --output-dir=doc/libgweather-4.0 \
	--no-namespace-dir \
	--content-dir=/hack/objdirs/pobj/libgweather4-4.0.0/libgweather-4.0.0/doc \
	libgweather/GWeather-4.0.gir
WARNING: Unable to process dot data: b'Fontconfig error:
No writable cache directories\nFontconfig error: No writable cache directories\n'

This happens if there's an empty directory under one of the fontconfig paths
(most likely /usr/local/share/fonts/) which probably happens during dpb(1)
junking (which is why it doesn't happen in every bulks).

Index: gidocgen/config.py
--- gidocgen/config.py.orig
+++ gidocgen/config.py
@@ -161,9 +161,7 @@ class GIDocConfig:
 
     @property
     def show_class_hierarchy(self):
-        if utils.find_program('dot') is None:
-            return False
-        return self.theme.get('show_class_hierarchy', False)
+        return False
 
     def source_link(self, *args):
         (filename, line) = args[0]
